start.sh

```diff
@@ -16,8 +17,8 @@ if [ "$role" = "app" ]; then
 
 elif [ "$role" = "queue" ]; then
 
-    echo "Running the queue..."
-    php /var/www/html/artisan queue:work --verbose --tries=5 --timeout=90
+    echo "Running the $queue_name queue..."
+    php /var/www/html/artisan queue:work --verbose --tries=5 --timeout=90 --queue=$queue_name
```

Comment:
```suggestion
    php /var/www/html/artisan queue:work --verbose --tries=5 --timeout=90 --queue="$queue_name"
```

most likely won't matter but I've been bitten by escaping so many times, I can't not recommend it :D